examples/boolean_org_exclude/README.md

```diff
@@ -6,25 +6,14 @@ If you wanted to set the policy on a folder level and selectively disable it on
 
 ```hcl
 module "folder-disable-serial-port-access-enforce-with-excludes" {
-  source            = "github.com/terraform-google-modules/terraform-google-org-policy"
-
+  source      = "github.com/terraform-google-modules/terraform-google-org-policy"
   folder_id   = "folders/my-folder-id"
   constraint  = "compute.disableSerialPortAccess"
-  enforce     = "true"
+  enforce     = true
   policy_type = "boolean"
-
   exclude_projects = ["my-project"]
 }
 ```
 
 [^]: (autogen_docs_start)
```

Comment:
These tags must be updated for the 0.12 documentation generation tool.

Reply:
Yes! Thank you! Fixed && added into checklist